84.md

```diff
@@ -0,0 +1,98 @@
+NIP-84
+======
+
+Highlights
+----------
+
+`draft` `optional` `author:pablof7z`
+
+This NIP defines `kind:9802`, a "highlight" event, to signal content a user finds valuable.
+
+## Format
+The `.content` of these events is the highlighted portion of the text.
+
+`.content` might be empty for highlights of non-text based media (e.g. NIP-94 audio/video).
+
+### References
+Events SHOULD tag the source of the highlight, whether nostr-native or not.
+`a` or `e` tags should be used for nostr events and `r` tags for URLs.
+
+When tagging a URL, clients generating these events SHOULD do a best effort of cleaning the URL from trackers
+or obvious non-useful information from the query string.
```

Comment:
Maybe it is a good idea to say that `r` urls should remove trailing slash so to ease querying.
So https://footstr.com/zapvertise/ -> https://footstr.com/zapvertise
And https://example.com/?article=987 -> https://example.com?article=987

Reply:
And also normalize the query strings?

Reply:
Good idea. Remove port if 80 (http) or 443 (https) only.